script/build.js

```diff
@@ -101,7 +116,11 @@ if (options.buildtype === 'production') {
   ignoreList.push('profile/*');
   ignoreList.push('auth/*');
   ignoreList.push('education/apply-for-education-benefits-new');
+} else if (options.buildtype === 'staging') {
+  ignoreList.push('education/apply-for-education-benefits/application.md');
```

Comment:
@akainic - I had this wrong, sorry for the confusion. The update should clear this up...